x-pack/filebeat/module/aws/cloudtrail/config/file.yml

```diff
@@ -4,6 +4,9 @@ paths:
   - {{$path}}
     {{ end }}
 exclude_files: [".gz$"]
+tags: {{.tags | tojson}}
+publisher_pipeline.disable_host: {{ inList .tags "cloud" }}
```

Comment:
overall makes sense to me, but just wondering--shouldn't this follow the same logic as the `add_host_metadata` processor (check `cloud` || `forwarded`)? Just wondering since it'd mean in 8.x we could collapse this down and just standardize on "tag with forwarded to avoid the host enrichment"